.aspell.en.pws

```diff
@@ -81,4 +81,5 @@ XDCR
 Eventing
 Analytics
 vBucket
-vBuckets
\ No newline at end of file
+vBuckets
+virtualised
```

Comment:
Put a completely blank line at the end of this file, so we don't get bogus diffs when someone adds a new word